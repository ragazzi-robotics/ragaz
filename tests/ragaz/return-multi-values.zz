def add(x: int, y: int) -> int:
    return x + y


def pick_one() -> int:
    return 3


def pick_three() -> tuple<int, int, int>:
    return 5, 6, 7


def main():
    print(add(3, 4))

    var a = pick_one()
    print(a)

    var b, c, d = pick_three()
    print(b)
    print(c)
    print(d)
