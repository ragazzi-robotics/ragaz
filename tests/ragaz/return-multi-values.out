7
3
5
6
7
